jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -127,12 +130,14 @@ public void setContainerCmd(List<String> cmd) {
     config.Cmd = cmd;
   }
 
-  public void setContainerExposedPorts(List<String> exposedPorts) {
+  public void setContainerExposedPorts(List<PortsWithProtocol> exposedPorts) {
     // TODO: Do this conversion somewhere else
     ImmutableSortedMap.Builder<String, Map<?, ?>> result =
         new ImmutableSortedMap.Builder<>(String::compareTo);
-    for (String port : exposedPorts) {
-      result.put(port, Collections.emptyMap());
+    for (PortsWithProtocol port : exposedPorts) {
+      for (int i = port.getMinPort(); i <= port.getMaxPort(); i++) {
```

Comment:
We want to avoid single letter variables. Something like `port` would be better.